src/server/sv_utils.ts

```diff
@@ -55,6 +53,17 @@ export class ServerUtils {
       });
     });
   }
+
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  private uuidV4(options?: any, buf?: any, offset?: any): string {
+    return "xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx".replace(/[xy]/g, function (c) {
+      const r = (Math.random() * 16) | 0,
+        v = c == "x" ? r : (r & 0x3) | 0x8;
+      return v.toString(16);
+    });
+  }
+
```

Comment:
Honestly considering this is probably one of the slowest parts of execution, we may as well reduce the uuid length as a chance of collision during promise resolution is somewhat low, maybe something like a length of 4, but I am open to thoughts. This works fine though.

Reply:
I added this because it's currently broken, as uuid isn't a dependency and it doesn't compile nicely for node in cfx as it doesn't support `require` (hence the change to esnext).

Doesn't have to be `uuid` does it?
Could do something like this (used in another resource already)

```js
/**
 * Creates a unique ID, such as `notif-0-2ammqh9cevf`
 *
 * @return {*} Unique ID
 */
function uid(prefix) {
  return `${prefix.trim()}-${(idCounter++).toString(36)}-${Math.floor(Math.random() * Number.MAX_SAFE_INTEGER).toString(36)}`;
}

```

Reply:
> a chance of collision during promise resolution is somewhat low

I don't know about this library. But I've had promise collision many many times with `qb-core`'s `TriggerCallback which is why I'm here. Might not be as low as you think, especially if you have 2 resources calling the same export at the same time (on startup).